napari/plugins/pypi.py

```diff
@@ -71,7 +75,77 @@ def conda_map() -> Dict[PyPIname, Optional[str]]:
         return json.load(resp)
 
 
-def iter_napari_plugin_info() -> Iterator[Tuple[PackageMetadata, bool]]:
+@lru_cache
+def pypi_package_versions(package_name: str) -> List[str]:
+    """Get available versions of a package on pypi.
+
+    Parameters
+    ----------
+    package_name : str
+        Name of the package.
+
+    Returns
+    -------
+    list
+        Versions available on pypi.
+    """
+    url = f"https://pypi.org/simple/{package_name}"
+    html = requests.get(url).text
+    package_name_ = package_name.replace('-', '_')
+    ret1 = re.findall(f">{package_name}-(.+).tar", html)
+    ret2 = re.findall(f">{package_name_}-(.+).tar", html)
+    return ret1 + ret2
+
+
+@lru_cache
+def conda_package_data(
```

Comment:
`pypi.py` no longer seems like a good name now that we're also fetching conda information. I dunno what a better name is though... `all_plugins.py` or something?